homeassistant/components/rest/notify.py

```diff
@@ -168,10 +166,15 @@ def _data_template_creator(value):
                     return {
                         key: _data_template_creator(item) for key, item in value.items()
                     }
+                if isinstance(value, str):
```

Comment:
It can be any value, not just a string.
We should check if it isn't a template instead.

Reply:
done